### PR TITLE
Rename placement group strategy

### PR DESCRIPTION
**Pull request by Ubun1**
**What this PR does / why we need it**:
- rename ```distribute``` -> ```spread``` strategy due to latest croc cloud [release](http://docs.website.cloud.croc.ru/en/api/ec2/placements/CreatePlacementGroup.html)

**Which issue(s) this PR fixes**:
- None

**Special notes for your reviewer**:
- None

### CODE REVIEW

File: cases/aws_placement_group/main.tf [hunk 1]
```diff
@@ -2,5 +2,5 @@ resource "aws_placement_group" "test_placement_group" {
   name = "test_placement_group"
 
   # NOTE: the only supported value for 'strategy' attribute is 'distribute'
```
Comment (pkulev) [inline cases/aws_placement_group/main.tf]: Forgot commentary.

**Review by pkulev — CHANGES_REQUESTED** [review]

**Comment by pkulev** [comment]
Also check README, https://github.com/C2Devel/terraform-examples/blob/master/cases/aws_placement_group/README.rst

File: cases/aws_placement_group/main.tf (hunk 1 above)
Comment (Ubun1) [reply cases/aws_placement_group/main.tf]: thx!

**Comment by Ubun1** [comment]
@pkulev more comments ?

**Review by pkulev — APPROVED** [review]
It's OK.

**Pull request merged**